test/test_iris.py

```diff
@@ -0,0 +1,45 @@
+'''
+ * Copyright (C) 2021  University of Alberta
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+'''
+'''
+ * @file test_iris.py
+ * @author Jenish Patel
+ * @date 2022-06-21
+'''
+
+'''  to run > yarn test_iris -I uart -d /dev/ttyUSB0 '''
+
+import numpy as np
+
+import sys
+from os import path
+sys.path.append("./test")
+
+from testLib import testLib as test
+
+import time
+
+test = test() #call to initialize local test class
+
+def testAllCommandsToOBC():
```

Comment:
Should this have a more descriptive name? Or even break the tests up in to one specific test per command

Reply:
I have another branch that has the testing stuff, I will have a seperate pr for this.  I will discard this file